modules/ept-users/routes/removeInvitation.js

```diff
@@ -27,7 +27,9 @@ module.exports = {
     // remove invitation
     var email = request.payload.email;
     var promise = request.db.users.removeInvite(email)
-    .then(function() { return { message: 'Invitation Removed.' }; });
+    .then(function() { return { message: 'Invitation Removed.' }; })
```

Comment:
Should all success instances return a similar message?  or should they just return nothing?

Will discuss.